crates/astria-sequencer/src/app/test_utils.rs

```diff
@@ -223,20 +223,21 @@ pub(crate) fn mock_tx(
     signer: &SigningKey,
     rollup_name: &str,
 ) -> Arc<SignedTransaction> {
-    let tx = UnsignedTransaction {
-        params: TransactionParams::builder()
-            .nonce(nonce)
-            .chain_id("test")
-            .build(),
-        actions: vec![
+    let tx = UnsignedTransaction::new(
+        vec![
             SequenceAction {
                 rollup_id: RollupId::from_unhashed_bytes(rollup_name.as_bytes()),
                 data: Bytes::from_static(&[0x99]),
                 fee_asset: denom_0(),
             }
             .into(),
         ],
-    };
+        TransactionParams::builder()
+            .nonce(nonce)
+            .chain_id("test")
+            .build(),
+    )
+    .expect("Failed to build unsigned transaction");
```

Comment:
```suggestion
    .expect("can build unsigned transaction");
```

Reply:
The semantics of @noot suggestion are better, but the new statement is somewhat vacuous. But since you want to create `mock_tx` it's clear from the context that you probably expect this to work as inputs to other tests:
```suggestion
    .unwrap();
```